launch_ros/launch_ros/actions/composable_node_container.py

```diff
@@ -63,8 +63,7 @@ def parse(cls, entity: Entity, parser: Parser):
         composable_nodes = entity.get_attr(
             'composable_node', data_type=List[Entity], optional=True)
         if composable_nodes is not None:
-            parsed_tuples = [ComposableNode.parse_composable_node(
-                parser, entity) for entity in composable_nodes]
+            parsed_tuples = [ComposableNode.parse(parser, entity) for entity in composable_nodes]
             kwargs['composable_node_descriptions'] = [t[0](**t[1]) for t in parsed_tuples]
```

Comment:
@kenji-miyake nit: something like
```suggestion
            kwargs['composable_node_descriptions'] = [cls(**attr) for cls, attr in parsed_tuples]
```
would be easier to read. Same elsewhere.

Reply:
Thank you, it's so nice!

Reply:
Changed as above in dad664d.